system/ee/ExpressionEngine/Addons/structure/addon.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "Structure",
     "shortname": "structure",
-    "version": "6.0.0",
+    "version": "6.1.1",
```

Comment:
Why is the version going to 6.1.1 and not 6.1.0? (the PR for  version with support for conditional access is setting 6.1.0)

Reply:
I'll update that to 6.1.0.